docs/en/maintain/cli.md

```diff
@@ -131,19 +131,19 @@ DelReplica ok
 DelReplica ok
 ```
 
-### migrate
+### Migrate
 
 Replicas migration
 
 Command format: `migrate src_endpoint table_name pid_group des_endpoint`
 
 * src\_endpoint: The endpoint of the node that needs to be checked out
-* table\_name: the table name
-* pid\_group:  the collection of shard IDs. There can be the following situations
-    * a single shard
+* table\_name: The table name
```

Comment:
this is the description for a specific term, by convention we also no need to capitalize the first word, please help change them back.